src/components/BankLink.tsx

```diff
@@ -0,0 +1,13 @@
+import { BankData } from "../../types";
+
+export default function Banklink(props: { data: BankData }) {
```

Comment:
Let's make this more type-safe with a Props Interface e.g. BankLinkProps and then add the BankData!